index.js

```diff
@@ -28,7 +28,14 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  let newMovie = [];
+  for(let i = 0; i < movies.length; i++){
+    newMovie.push(movies[i].title);
+  }
+  return newMovie;
```

Comment:
Code looks great!  I would recommend something more like `allTitles` or `movieTitles` for a variable name.  That way when I read it I know exactly what its purpose is